android/capacitor/src/main/java/com/getcapacitor/MessageHandler.java

```diff
@@ -123,12 +122,12 @@ public void sendResponseMessage(PluginCall call, PluginResult successResult, Plu
 
             boolean isValidCallbackId = !call.getCallbackId().equals(PluginCall.CALLBACK_ID_DANGLING);
             if (isValidCallbackId) {
-                if (WebViewFeature.isFeatureSupported(WebViewFeature.WEB_MESSAGE_LISTENER) && javaScriptReplyProxy != null) {
+                if (bridge.getConfig().isUsingLegacyBridge()) {
```

Comment:
I had to use an extra if because Android Studio would complain if I put this check with `!` on the `isFeatureSupported` check on the `postMessage` line